src/content/es/2019/javascript.md

```diff
@@ -0,0 +1,439 @@
+          <td class="numeric">5.93%</td>
+        </tr>
+        <tr>
+          <td>Slick</td>
+          <td class="numeric">5.53%</td>
+          <td class="numeric">5.24%</td>
+        </tr>
+        <tr>
+          <td>Moment.js</td>
+          <td class="numeric">4.92%</td>
+          <td class="numeric">4.29%</td>
+        </tr>
+        <tr>
+          <td>Underscore.js</td>
+          <td class="numeric">4.20%</td>
+          <td class="numeric">3.82%</td>
+        </tr>
+        <tr>
+          <td>prettyPhoto</td>
+          <td class="numeric">2.89%</td>
+          <td class="numeric">3.09%</td>
+        </tr>
+        <tr>
+          <td>Select2</td>
+          <td class="numeric">2.78%</td>
+          <td class="numeric">2.48%</td>
+        </tr>
+        <tr>
+          <td>Lodash</td>
+          <td class="numeric">2.65%</td>
+          <td class="numeric">2.68%</td>
+        </tr>
+        <tr>
+          <td>Hammer.js</td>
+          <td class="numeric">2.28%</td>
+          <td class="numeric">2.70%</td>
+        </tr>
+        <tr>
+          <td>YUI</td>
+          <td class="numeric">1.84%</td>
+          <td class="numeric">1.50%</td>
+        </tr>
+        <tr>
+          <td>Lazy.js</td>
+          <td class="numeric">1.26%</td>
+          <td class="numeric">1.56%</td>
+        </tr>
+        <tr>
+          <td>Fingerprintjs</td>
+          <td class="numeric">1.21%</td>
+          <td class="numeric">1.32%</td>
+        </tr>
+        <tr>
+          <td>script.aculo.us</td>
+          <td class="numeric">0.98%</td>
+          <td class="numeric">0.85%</td>
+        </tr>
+        <tr>
+          <td>Polyfill</td>
+          <td class="numeric">0.97%</td>
+          <td class="numeric">1.00%</td>
+        </tr>
+        <tr>
+          <td>Flickity</td>
+          <td class="numeric">0.83%</td>
+          <td class="numeric">0.92%</td>
+        </tr>
+        <tr>
+          <td>Zepto</td>
+          <td class="numeric">0.78%</td>
+          <td class="numeric">1.17%</td>
+        </tr>
+        <tr>
+          <td>Dojo</td>
+          <td class="numeric">0.70%</td>
+          <td class="numeric">0.62%</td>
+        </tr>
+      </tbody>
+    </table>
+	<figcaption>Figura 11. Principales bibliotecas de JavaScript en computadoras de escritorio y dispositivos móviles.</figcaption>
+</figure>
+
+[jQuery](https://jquery.com/), la biblioteca JavaScript más popular jamás creada, se utiliza en el 85.03% de las páginas de escritorio y el 83.46% de las páginas móviles. El advenimiento de muchas API y métodos del navegador, tales como [Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) y [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector), estandarizaron gran parte de la funcionalidad proporcionada por la biblioteca en una forma nativa. Aunque la popularidad de jQuery puede parecer estar disminuyendo, ¿por qué todavía se usa en la gran mayoría de la web?
```

Comment:
Removing `en-US` from MDN pages that have an equivalent page available in Spanish.

```suggestion
[jQuery](https://jquery.com/), la biblioteca JavaScript más popular jamás creada, se utiliza en el 85.03% de las páginas de escritorio y el 83.46% de las páginas móviles. El advenimiento de muchas API y métodos del navegador, tales como [Fetch](https://developer.mozilla.org/docs/Web/API/Fetch_API) y [querySelector](https://developer.mozilla.org/docs/Web/API/Document/querySelector), estandarizaron gran parte de la funcionalidad proporcionada por la biblioteca en una forma nativa. Aunque la popularidad de jQuery puede parecer estar disminuyendo, ¿por qué todavía se usa en la gran mayoría de la web?
```